src/tds/log.c

```diff
@@ -56,6 +56,10 @@ int tds_g_append_mode = 0;
 static char *g_dump_filename = NULL;
 /** Tell if TDS debug logging is turned on or off */
 int tds_write_dump = 0;
+#ifdef TLS
+TLS
+#endif
+int tdsdump_elided = 0;
```

Comment:
If TLS is not defined this would be a global like `tds_write_dump`

Reply:
Yes, modulo the usual visibility constraints from `pushvis.h`.  That's a necessary evil, given the need to coordinate between `log.c` and `login.c`.